2023-06-14 13:07:39 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
  GATYA_ID VARCHAR(20) NOT NULL
  FOREIGN[*] KEY(GATYA_ID)           
  REFERENCES GATYA(GATYA_ID)                  

)"; 期待されるステートメント "AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
  GATYA_ID VARCHAR(20) NOT NULL
  FOREIGN[*] KEY(GATYA_ID)           
  REFERENCES GATYA(GATYA_ID)                  

)"; expected "AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
  GATYA_ID VARCHAR(20) NOT NULL
  FOREIGN KEY(GATYA_ID) /* 外部キー */
  REFERENCES GATYA(GATYA_ID) /* ガチャテーブル.部署番号*/

) [42001-200]
2023-06-14 13:15:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
  GATYA_ID VARCHAR(20) NOT NULL
  FOREIGN[*] KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)                  
)"; 期待されるステートメント "AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
  GATYA_ID VARCHAR(20) NOT NULL
  FOREIGN[*] KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)                  
)"; expected "AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
  GATYA_ID VARCHAR(20) NOT NULL
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID) /* ガチャテーブル.部署番号*/
) [42001-200]
2023-06-14 13:15:51 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "GATYA" が見つかりません
Table "GATYA" not found; SQL statement:
CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID) 
) [42102-200]
2023-06-14 13:21:38 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
)[*]"; 期待されるステートメント "identifier"
Syntax error in SQL statement "CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
)[*]"; expected "identifier"; SQL statement:
CREATE TABLE USER (
  STAFF_ID INT PRIMARY KEY AUTO_INCREMENT,
  PASSWORD VARCHAR(20) NOT NULL,
  NAME VARCHAR(20) NOT NULL,
  ROLE INT(1),
  Q_POINT INT(99999),
  QUIZ BOOLEAN DEFAULT NULL,
  ANSWER_TIME DATETIME NOT NULL,
  ASGR_TICKET INT(10) NOT NULL,
  SMRI_TICKET INT(10) NOT NULL,
  SDS_TICKET INT(10) NOT NULL, 
) [42001-200]
2023-06-14 13:28:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
  WORK_DATE[*] DATE NOT NULL,
)"; 期待されるステートメント "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"
Syntax error in SQL statement "CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
  WORK_DATE[*] DATE NOT NULL,
)"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
  WORK_DATE DATE NOT NULL,
) [42001-200]
2023-06-14 13:28:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
  WORK_DATE[*] DATE NOT NULL
)"; 期待されるステートメント "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"
Syntax error in SQL statement "CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
  WORK_DATE[*] DATE NOT NULL
)"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
  WORK_DATE DATE NOT NULL
) [42001-200]
2023-06-14 13:28:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "STAFF_ID" が見つかりません
Column "STAFF_ID" not found; SQL statement:
CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) ,
  WORK_DATE DATE NOT NULL
) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:216)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-14 13:50:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
CREATE TABLE WORKING (
  WORK_ID INT PRIMARY KEY AUTO_INCREMENT,
  WORK_START DATETIME NOT NULL,
  WORK_END DATETIME NOT NULL,
  WORK_STYLE VARCHAR(20) NOT NULL,
  STAFF_ID INT PRIMARY KEY,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) ,
  WORK_DATE DATE NOT NULL
) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.CommandWithColumns.setPrimaryKey(CommandWithColumns.java:148)
	at org.h2.command.ddl.CommandWithColumns.addConstraintCommand(CommandWithColumns.java:49)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8474)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-14 14:08:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE TASK (
  TASK_ID INT PRIMARY KEY AUTO_INCREMENT,
  TASK_THREAD VARCHAR(200) NOT NULL,
  CALENDAR DATETIME NOT NULL,
  STAFF_ID INT ,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) ,
)[*]"; 期待されるステートメント "identifier"
Syntax error in SQL statement "CREATE TABLE TASK (
  TASK_ID INT PRIMARY KEY AUTO_INCREMENT,
  TASK_THREAD VARCHAR(200) NOT NULL,
  CALENDAR DATETIME NOT NULL,
  STAFF_ID INT ,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) ,
)[*]"; expected "identifier"; SQL statement:
CREATE TABLE TASK (
  TASK_ID INT PRIMARY KEY AUTO_INCREMENT,
  TASK_THREAD VARCHAR(200) NOT NULL,
  CALENDAR DATETIME NOT NULL,
  STAFF_ID INT ,
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) ,
) [42001-200]
2023-06-14 15:16:22 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE USER ADD GATYA_ID INT,[*]
FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)"
Syntax error in SQL statement "ALTER TABLE USER ADD GATYA_ID INT,[*]
FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)"; SQL statement:
ALTER TABLE USER ADD GATYA_ID INT,
FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID) [42000-200]
2023-06-14 15:17:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE USER ADD GATYA_ID VARCHAR
FOREIGN[*] KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)"
Syntax error in SQL statement "ALTER TABLE USER ADD GATYA_ID VARCHAR
FOREIGN[*] KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)"; SQL statement:
ALTER TABLE USER ADD GATYA_ID VARCHAR
FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID) [42000-200]
2023-06-14 15:17:56 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE USER ADD GATYA_ID VARCHAR,[*]
FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)"
Syntax error in SQL statement "ALTER TABLE USER ADD GATYA_ID VARCHAR,[*]
FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID)"; SQL statement:
ALTER TABLE USER ADD GATYA_ID VARCHAR,
FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID) [42000-200]
2023-06-14 15:22:39 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE USER ADD FOREIGN KEY GATYA_ID[*](GATYA_ID) REFERENCES GATYA (GATYA_ID)"; 期待されるステートメント "("
Syntax error in SQL statement "ALTER TABLE USER ADD FOREIGN KEY GATYA_ID[*](GATYA_ID) REFERENCES GATYA (GATYA_ID)"; expected "("; SQL statement:
ALTER TABLE USER ADD FOREIGN KEY GATYA_ID(GATYA_ID) REFERENCES GATYA (GATYA_ID) [42001-200]
2023-06-14 15:24:18 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "GATYA_ID" が見つかりません
Column "GATYA_ID" not found; SQL statement:
ALTER TABLE USER ADD FOREIGN KEY (GATYA_ID) REFERENCES GATYA (GATYA_ID) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:216)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-14 15:34:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "[*]【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR(20),
  STAFF_ID INT,
  PRIMARY KEY(GATYA_ID,STAFF_ID),
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
)"
Syntax error in SQL statement "[*]【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR(20),
  STAFF_ID INT,
  PRIMARY KEY(GATYA_ID,STAFF_ID),
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
)"; SQL statement:
【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR(20),
  STAFF_ID INT,
  PRIMARY KEY(GATYA_ID,STAFF_ID),
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
) [42000-200]
2023-06-14 15:35:57 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "[*]【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR,
  STAFF_ID INT,
  PRIMARY KEY(GATYA_ID,STAFF_ID),
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
)"
Syntax error in SQL statement "[*]【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR,
  STAFF_ID INT,
  PRIMARY KEY(GATYA_ID,STAFF_ID),
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
)"; SQL statement:
【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR,
  STAFF_ID INT,
  PRIMARY KEY(GATYA_ID,STAFF_ID),
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
) [42000-200]
2023-06-14 15:37:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "[*]【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR PRIMARY KEY,
  STAFF_ID INT PRIMARY KEY, 
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
)"
Syntax error in SQL statement "[*]【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR PRIMARY KEY,
  STAFF_ID INT PRIMARY KEY, 
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
)"; SQL statement:
【GATYAGETテーブルを作成するSQL文】
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR PRIMARY KEY,
  STAFF_ID INT PRIMARY KEY, 
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
) [42000-200]
2023-06-14 15:37:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
CREATE TABLE GATYAGET (
  GATYA_ID VARCHAR PRIMARY KEY,
  STAFF_ID INT PRIMARY KEY, 
  FOREIGN KEY(GATYA_ID) REFERENCES GATYA(GATYA_ID),
  FOREIGN KEY(STAFF_ID) REFERENCES USER(STAFF_ID) 
) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.CommandWithColumns.setPrimaryKey(CommandWithColumns.java:148)
	at org.h2.command.ddl.CommandWithColumns.addConstraintCommand(CommandWithColumns.java:49)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8474)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2023-06-14 15:50:29 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE WORD (
  WORD_ID INT PRIMARY KEY AUTO_INCREMENT,,[*]
  WORD_ITEM VARCHAR(100) NOT NULL,
  WORD_EX VARCHAR(1000) NOT NULL   
)"; 期待されるステートメント "identifier"
Syntax error in SQL statement "CREATE TABLE WORD (
  WORD_ID INT PRIMARY KEY AUTO_INCREMENT,,[*]
  WORD_ITEM VARCHAR(100) NOT NULL,
  WORD_EX VARCHAR(1000) NOT NULL   
)"; expected "identifier"; SQL statement:
CREATE TABLE WORD (
  WORD_ID INT PRIMARY KEY AUTO_INCREMENT,,
  WORD_ITEM VARCHAR(100) NOT NULL,
  WORD_EX VARCHAR(1000) NOT NULL   
) [42001-200]
2023-06-15 10:12:35 database: wrong user or password; user: "SA"
org.h2.message.DbException: ユーザ名またはパスワードが不正です
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: ユーザ名またはパスワードが不正です
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2023-06-15 10:12:42 database: wrong user or password; user: "SA"
org.h2.message.DbException: ユーザ名またはパスワードが不正です
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: ユーザ名またはパスワードが不正です
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2023-06-15 13:17:02 jdbc[6]: null
org.h2.message.DbException: アプリケーションにより閉じられていない接続がガベージコレクトされました
The connection was not closed by the application and is garbage collected [90018-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:245)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:186)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: アプリケーションにより閉じられていない接続がガベージコレクトされました
The connection was not closed by the application and is garbage collected [90018-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:622)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 16 more
